network/ssh/reachable.go

```diff
@@ -0,0 +1,216 @@
+// Copyright 2014 Canonical Ltd.
```

Comment:
This was actually moved from network/reachable.go, so I think the original copyright applies.